src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {

```

Comment:
we do not need Fruit class. 
int amount is not a fruit property - it is Storage responsibility